### PR TITLE
New package: SimonG4ELI.WorldMap version 1.4.0

### LINKED ISSUE #138630: [Package Request]: Simon's World Map
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.sdr-radio.com/world-map#Downloads
Publisher: Simon G4ELI
Package Name: Simon's World Map
Description: A free map of the world for radio aficionados.
Package Version: 1.4.0
Installer URL: https://www.dropbox.com/scl/fi/k8nzcys5n8pe1sc7zu7tm/Simon-s-World-Map-2024-02-04-19h24m.exe?rlkey=abskz0i1tyjgsbwlcw5pufxti&dl=1

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---
- Resolve: #138630

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138656)

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Just a note: The Dropbox url was already used for Beta version of SDR-Radio and it worked fine so it should not be a problem. :-) Simon's World Map is sadly not available for download on the CDN website, just via Dropbox or OneDrive.

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/s/SimonG4ELI/WorldMap/1.4.0/SimonG4ELI.WorldMap.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SimonG4ELI.WorldMap
+PackageVersion: 1.4.0
+InstallerType: nullsoft
+Installers:
+- InstallerUrl: https://www.dropbox.com/scl/fi/k8nzcys5n8pe1sc7zu7tm/Simon-s-World-Map-2024-02-04-19h24m.exe?rlkey=abskz0i1tyjgsbwlcw5pufxti&dl=1
+  Architecture: x86
+  InstallerSha256: C14CE4B64A2A374B96D207DA4F25B3968F11BBB753F72C5E4AAD9838C96758B4
```
Comment (Trenly) [inline manifests/s/SimonG4ELI/WorldMap/1.4.0/SimonG4ELI.WorldMap.installer.yaml:10]:
```suggestion
  InstallerSha256: C14CE4B64A2A374B96D207DA4F25B3968F11BBB753F72C5E4AAD9838C96758B4
  AppsAndFeaturesEntries:
    - DisplayVersion: 2024-02-04 19h24m
```
Required to match the package version for upgrade

File: manifests/s/SimonG4ELI/WorldMap/1.4.0/SimonG4ELI.WorldMap.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SimonG4ELI.WorldMap
+PackageVersion: 1.4.0
+PackageLocale: en-US
+Publisher: Simon G4ELI
```
Comment (Trenly) [inline manifests/s/SimonG4ELI/WorldMap/1.4.0/SimonG4ELI.WorldMap.locale.en-US.yaml:7]:
```suggestion
Publisher: Simon Brown, G4ELI
```
Should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

Could you add a PackageUrl?

(Automated response - build 729.)

**Comment by AsciiWolf** [comment]
Also feel free to add `PublisherUrl: https://www.dit-dit-dit.com`. ;-)

**Comment by Exorcism0666** [comment]
@stephengillie

**Comment by AsciiWolf** [comment]
Is there anything preventing this PR from being merged? It looks good to me.

**Comment by AsciiWolf** [comment]
Ping. :-)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**